.github/workflows/tests-on-pr-push-and-merge.yml

```diff
@@ -23,10 +23,6 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: 17
-    - name: Run linter
```

Comment:
why is linter check removed? is it because you added the new rule in spotless.gradle?

Reply:
Yes, the build step includes spotlessCheck on github CI or spotlessApply on local runs